zooniverse/front-end-monorepo
# lib-user: Refactor dev App to use PJC auth client

## Package
- lib-user

## Describe your changes
- refactor lib-user dev app to use PJC auth client (instead of PJC oauth)

### Notes
- there's a ResizeObserver error that I think is related to my LastPass browser extension and a sign-in field input width ([SO link](https://stackoverflow.com/a/76190200)). if someone without LastPass also gets this error I'll revisit for a fix, but if it is indeed just a LastPass browser extension issue I'm going to leave as is.

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX? n/a
- What user actions should my reviewer step through to review this PR?
  - locally from lib-user, run `yarn dev`
  - sign in/out at https://local.zooniverse.org:8080/ (with staging user)
- Which storybook stories should be reviewed? n/a
- Are there plans for follow up PR’s to further fix this bug or develop this feature?
  - import lib-react-components header and footer for dev app (with warnings not in app-root and routing won't work)
  - change lib-user from being auth-client agnostic, to expecting PJC `auth` by:
    - refactor lib-user components/hooks to use PJC auth
    - remove authClient as a prop passed into lib-user
    - update README/documentation accordingly

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## Refactoring
